docs/install.rst

```diff
@@ -412,12 +413,16 @@ dependencies, including:
 * `numpydoc <https://numpydoc.readthedocs.io>`_ - an extension to parse
   docstrings in NumPyDoc format
 * `pillow <https://pillow.readthedocs.io>`_ - used in one of the examples
+* `Graphviz <http://www.graphviz.org>`_ - generate inheritance graphs (ONLY
+  available in conda package)
+
+.. Note::
```

Comment:
Does this need to be a separate note? Maybe can be sentences on the Graphviz listing instead?

Reply:
I just know that I did not understand this for a while and could not figure out why I was getting thousands of lines of output from building the docs.  So I'm writing these docs based on my former-self experience, which might match others.  Or not?